entgql/internal/todo/todo_test.go

```diff
@@ -640,6 +640,118 @@ func (s *todoTestSuite) TestPaginationFiltering() {
 	})
 }
 
+func (s *todoTestSuite) TestEdgesFiltering() {
```

Comment:
I think we can move this test case to the [`NestedPagination`/`NestedConnection`](https://github.com/ent/contrib/blob/master/entgql/internal/todo/todo_test.go#L1266) one.

Also, in this test case, you don't really test the `Query<E>` method, because the edges are resolved/loaded by the fields-collector (using eager-loading). 

@amrnt, how do you use this API? Do you call these methods directly?

Reply:
OK, I see now. It's called `childrenLevel1` and that's why it's not loaded. 

I still want to give it a look before merging it.